packages/lambdapi/lambdapi.2.1.0/opam

```diff
@@ -48,13 +48,12 @@ depends: [
   "sedlex" {>= "2.2"}
   "alcotest" {with-test}
   "alt-ergo" {with-test}
-  "alt-ergo" {<= "2.4.0"}
   "bindlib" {>= "5.0.1"}
```

Comment:
@fblanqui You'll also need to change that to an equality. The release of Bindlib 6.0.0 (see https://github.com/ocaml/opam-repository/pull/20808) is gonna break Lambdapi otherwise. In general, it is not guaranteed that libraries are backward compatible, so it is a good idea to keep dependencies strict.

Reply:
I actually did the constraining myself in https://github.com/ocaml/opam-repository/pull/20808. (Also see https://github.com/Deducteam/lambdapi/issues/852.)

Reply:
Thanks @rlepigre . This change is fine for me.